### PR TITLE
C2 amr a raouf c1 s20 admin populate database by importing csv

### PR DESCRIPTION
**Pull request by AmrARaouf**
[ISSUE NUMBER #82]
@Omar-Hossam 
@karimngr 
@Mirna-Yacout 
@zoheiry

### CODE REVIEW

**Comment by Mirna-Yacout** [comment]
@AmrARaouf the arrow returning message shouldnt it be dashed?

**Comment by m0hamed** [comment]
the issue number is wrong ya @AmrARaouf. Also why is there 242 files changed by your commit

**Comment by m0hamed** [comment]
you used git -A didnt you :-1:

**Comment by AmrARaouf** [comment]
@m0hamed no I used git add (dir) like normal people I have no idea why I have 242 changed files
and I noticed the issue number a while ago, all fixed now

**Comment by AmrARaouf** [comment]
@m0hamed ok I think I know what's wrong I'll ask you how to fix it tomorrow

**Comment by omar-mohyeldin** [comment]
@AmrARaouf file config/routes.rb line 71, check rails code conventions

**Comment by omar-mohyeldin** [comment]
@AmrARaouf file app/models/synonym.rb, repull my file from my branch or edit it yourself because it's not up-to-date to avoid future conflicts

**Comment by omar-mohyeldin** [comment]
@AmrARaouf file app/models/keyword.rb , check rails code conventions and modify

**Comment by omar-mohyeldin** [comment]
@AmrARaouf file app/models/category.rb , check rails code conventions and modify

**Comment by AmrARaouf** [comment]
@Omar-Hossam are you sure the synonym.rb file fixing is necessary? conflicts will be resolved when we all merge tomorrow

**Comment by AmrARaouf** [comment]
also line 71 in routes.rb is a commented line

**Comment by AmrARaouf** [comment]
@Mirna-Yacout check my UML now keda

**Comment by omar-mohyeldin** [comment]
tb 5alas forget about the synonym.rb updating
w nvm about the route.rb, tahneeg 2a5er 2l leil

**Comment by omar-mohyeldin** [comment]
@AmrARaouf ml 2a5er, modify all your .rb files to satisfy rails code conventions stated on mailing list

**Comment by Mirna-Yacout** [comment]
reviewed (Y)

File: arability/app/controllers/search_controller.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class SearchController < ApplicationController
```
Comment (zoheiry) [inline arability/app/controllers/search_controller.rb:1]: Please add the #author to the documentation so is will follow the template we agreed on

File: arability/app/models/synonym.rb [hunk 2]
```diff
@@ -1,4 +1,35 @@
 class Synonym < ActiveRecord::Base
   belongs_to :keyword
   attr_accessible :approved, :name
+
```
Comment (zoheiry) [inline arability/app/models/synonym.rb:8]: author should be added here also @AmrARaouf

**Comment by omar-mohyeldin** [comment]
@AmrARaouf  only the admin_controller.rb file has been modified, what about the rest of the .rb files? please modify them

**Comment by AmrARaouf** [comment]
the rest of the .rb files are not mine and will be fixed during conflicts, approve me there's nothing wrong with my code

**Comment by omar-mohyeldin** [comment]
@AmrARaouf reviewed, solve conflicts to merge ;)

**Pull request merged**